components/profile/UserNameTitle/index.tsx

```diff
@@ -14,12 +14,13 @@ const USER_NAME_TITLE = () => gql`
 function UserNameTitle() {
   const { loading, error, data } = useQuery(USER_NAME_TITLE())
 
-  if (loading || error)
+  if (loading)
     return (
       <div>
         <Loader open={loading} error={error} />
       </div>
     )
+  if (error) return <div className="max-w-3xl mx-auto">{JSON.stringify(error, null, 2)}</div>
```

Comment:
revert this to show error within Loader itself. That will keep the code concise in all other page loads as well